ui.py

```diff
@@ -1,25 +1,26 @@
-#!/usr/bin/env python3
-# -*- coding: utf-8 -*-
+#! /usr/bin/env python3
+# coding=utf-8
+
 
 import os
 import sys
 
-# If we're on Windows, use the included compiled DLLs.
-if sys.platform == "win32":
-    os.environ["PYSDL2_DLL_PATH"] = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'libs')
-
-from sdl2 import *
-from sdl2.sdlttf import *
 import sdl2.ext
 
-from const import Colors, WindowSize
-
+from sdl2 import *  # Bad idea to import like this!
```

Comment:
Hi! pyart here. :)  I wasn't able to run your code though, but I'm just saying that star imports are bad. Just import from sdl2 whatever you need, because you import all the unnecessary things that the sdl2 module/library holds. That way unwanted things are lurking in your code. Do star imports only if the authors say it's safe to do star imports of their library, in which case they would explicitly define the __all__ atribute in their library.